src/homepage/cc-doc-card.js

```diff
@@ -0,0 +1,142 @@
+import '../atoms/cc-img.js';
+import { css, html, LitElement } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { fakeString } from '../lib/fake-strings.js';
+import { i18n } from '../lib/i18n.js';
+import { defaultThemeStyles } from '../styles/default-theme.js';
+import { skeletonStyles } from '../styles/skeleton.js';
+import { linkStyles } from '../templates/cc-link.js';
+
+const SKELETON_INFO = {
+  title: fakeString(6),
+  description: fakeString(110),
+};
+
+/**
+ * A component displaying basic information of a product with a link to redirect to its documentation in a card.
+ *
+ * @cssdisplay grid
+ */
+export class CcDocCard extends LitElement {
+
+  static get properties () {
+    return {
+      description: { type: String },
+      icons: { type: Array },
+      link: { type: String },
+      title: { type: String },
+    };
+  }
+
+  constructor () {
+    super();
+
+    /** @type {string|null} Sets the description of the documentation card. */
+    this.description = null;
+
+    /** @type {string[]} Sets the icon of the documentation card. */
+    this.icons = null;
+
+    /** @type {string|null} Sets the link of the documentation card. */
+    this.link = null;
+
+    /** @type {string|null} Sets the title of the documentation card.  */
+    this.title = null;
+  }
+
+  render () {
+
+    const skeleton = (this.icons == null || this.title == null || this.description == null || this.link == null);
+    const title = this.title ?? SKELETON_INFO.title;
+    const description = this.description ?? SKELETON_INFO.description;
+
+    return html`
+        <div class="images">
+          ${skeleton ? html`
+            <cc-img></cc-img>
+          ` : ''}
+          ${!skeleton ? html`
+            ${this.icons.map((icon) => html`
+              <cc-img src=${icon}></cc-img>
```

Comment:
interesting case. These icons typically convey redundant information but in some cases like "Java", they convey additional info compared to the context.
In the example below, these images tell me that the doc is also related to "mvn" for instance.
![image](https://user-images.githubusercontent.com/100240294/168557950-723f8668-a338-4d4d-ba1d-ca4a26f4f00d.png)

The best solution (to avoid adding alt values that would be redundant in most cases) would be to add this piece of info in the description text but is it something we can do ?

Reply:
nevermind, we can assume these are just random, not exhaustive examples. Let's not overthink it.
It's good as it is ;)